src/creatures/combat/spells.cpp

```diff
@@ -630,6 +630,40 @@ void Spell::setWheelOfDestinyBoost(WheelSpellBoost_t boost, WheelSpellGrade_t gr
 	}
 }
 
+void Spell::getCombatDataAugment(std::shared_ptr<Player> player, CombatDamage &damage) {
+	if (!(damage.instantSpellName).empty()) {
+		std::vector<std::shared_ptr<Item>> equippedAugmentItems = player->getEquippedAugmentItems();
+		for (std::shared_ptr<Item> item : equippedAugmentItems) {
```

Comment:
Pass as a constant reference to avoid unnecessary copying
```suggestion
		const auto &equippedAugmentItems = player->getEquippedAugmentItems();
		for (const auto &item : equippedAugmentItems) {
```